NashTeamAlpha/BangazonWeb
# Subproducts description

## Pull Request Description

Shows a description of every subproduct type above the list of products that fit into that subproduct type.

## Steps to Test

1. git checkout subproduct
1. Navigate to a product type, then click on a subproduct type to see its description

## Link to Feature Ticket

#162
